src/common/store/index.ts

```diff
@@ -87,6 +93,8 @@ export const sagas = {
   cache: cacheSagas,
   collectionsError: collectionsErrorSagas,
   collections: collectionsSagas,
+  explore: explorePageSagas,
+  exploreCollections: exploreCollectionsSagas,
```

Comment:
Let hold off on moving the sagas because they have dependencies on AudiusBackend which won't work in RN yet. 

Can you add a comment below line 102 instead?